textattack/datasets/classification/ag_news.py

```diff
@@ -33,6 +33,8 @@ class AGNews(ClassificationDataset):
     
     """
     DATA_PATH = 'datasets/classification/ag_news.txt'
-    def __init__(self, offset=0):
+    def __init__(self, offset=0, shuffle=False):
         """ Loads a full dataset from disk. """
         self._load_classification_text_file(AGNews.DATA_PATH, offset=offset)
+        if shuffle:
```

Comment:
Two questions about this `shuffle` pattern:

1. can we move all the shuffle checks into `_load_classification_text_file` so the code isn't repeated?

2. will this shuffling work with `nlp` datasets?

Reply:
1. Done.
2. NLP dataset class already handles shuffling the same way.